src/cloud/project/project-patch.md

```diff
@@ -35,37 +29,165 @@ All patch file names must end with the `.patch` extension.
+╟────────────────┼─────────────────────────────────────────────────┼──────────┼─────────────┼─────────────────────────────────╢
+║ N/A            │ ../m2-hotfixes/MDVA_custom__2.3.5_ce.patch      │ Custom   │ N/A         │ Affected components:            ║
+║                │                                                 │          │             │  - magento/module-framework     ║
+╚════════════════╧═════════════════════════════════════════════════╧══════════╧═════════════╧═════════════════════════════════╝
+Magento 2 Enterprise Edition, version 2.3.5.0
+```
+
+The status table contains the following types of information:
+
+-  **Type**:
+
+   -  Optional—All individual patches from the MQP package and the [Magento Cloud Patches]({{ site.baseurl }}/cloud/project/project-patch.html) package are optional for {{ site.data.var.ee }} and {{ site.data.var.ce }} installations.
+   -  Required—All individual patches from the {{ site.data.var.mcp-prod }} package are required for Cloud customers.
+   -  Deprecated—The individual patch is marked as deprecated by Magento and we recommend reverting it if you have applied it. After you revert a deprecated patch, it will no longer be displayed in the status table.
+   -  Custom—All patches from the 'm2-hotfixes' directory.
+
+-  **Status**:
+
+   -  Applied—The patch has been applied.
+   -  Not applied—The patch has not been applied.
+   -  N/A—The status of the patch cannot be defined due to conflicts.
+
+-  **Details**:
+
+   -  Affected components—The list of affected Magento modules.
+   -  Required patches—The list of required patches (dependencies).
+   -  Recommended replacement—The patch that is a recommended replacement for a deprecated patch.
+
+## Apply a patch in a local environment
 
 You can apply patches manually in a local environment and test them before you deploy.
 
 {:.procedure}
-To apply {{site.data.var.ece}} patches manually:
+To apply patches in a local environment:
+
+1. Add the 'QUALITY_PATCHES' variable to the `.magento.env.yaml` file and list the required patches underneath.
+
+   ```yaml
+   stage:
+       build:
+           QUALITY_PATCHES:
+               - MCTEST-1002
+               - MCTEST-1003
+   ```
 
 1. From the project root, apply the patches.
 
    ```bash
    php ./vendor/bin/ece-tools patch
    ```
 
+   Patches will be applied in the following order:
+
+   -  Required patches
+   -  Optional individual patches
+   -  Custom patches from the `/m2-hotfixes` directory
+
 1. Clear the Magento cache.
 
    ```bash
    php ./bin/magento cache:clean
    ```
 
-   You can also clean the cache using the [Magento Admin Cache Management](http://docs.magento.com/m2/ee/user_guide/system/cache-management.html).
-
 1. Test the patches, make any necessary changes to custom patches.
 
+## Apply a patch in a Cloud environment
+
+{:.bs-callout-warning}
+We strongly recommend testing all patches in a staging or development environment before deploying to production.
```

Comment:
Integration, Staging, and Production environments are capitalized in Cloud. Also, it may be important to distinguish whether a test is sufficient in the Integration environment, since some services are not available.

```suggestion
We strongly recommend testing all patches in a Integration or Staging environments before deploying to the Production environment.
```